python/katana/example_data.py

```diff
@@ -11,8 +11,15 @@
 __all__ = ["get_input", "get_input_as_url"]
 
 
-def get_inputs_directory(*, invalidate=False) -> Path:
+def get_inputs_directory(*, invalidate=False, rel_path=None) -> Path:
     inputs_dir = None
+    if rel_path == "external/csv-datasets":
```

Comment:
I don't like special casing for specific relative paths. If we need to so that we might want to add a "data set name" parameter or something to select a specific data set that needs to be downloaded.

Reply:
Note that the plan is not to download data, but to use data available on gs